tests/testthat/test-interpolate.R

```diff
@@ -137,6 +137,7 @@ test_that("can interpolate a matrix, piecewise linear", {
 
 
 test_that("can interpolate a matrix, spline", {
+  set.seed(1)
```

Comment:
Unrelated change leading to stochastic test failures. A similar seed is present in the single case so I guess we've seen this before